<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="ArchiveList">
      <property name="text">
       <string>Archive chat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Name :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Delete_this_chat">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLabel" name="FirstName_lastName">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>90</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name_last_name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>User Name:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRead">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>30</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
      <widget class="QLabel" name="UserName">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>User_</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SeeChat">
       <property name="geometry">
        <rect>
         <x>44</x>
         <y>20</y>
         <width>101</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Chat</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Archive_this_chat">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Archive</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NextButton">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BeforeButton">
      <property name="text">
       <string>Before</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="BackButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
